packages/asset-uploader/src/upload.ts

```diff
@@ -39,23 +38,24 @@ export const createUploadName = async (
    * than UPLOADING_STALE_TIMEOUT milliseconds ago
    **/
 
-  const count = await prisma.file.count({
-    where: {
-      OR: [
-        {
-          status: "UPLOADED",
-          isDeleted: false,
-          uploaderProjectId: projectId,
-        },
-        {
-          status: "UPLOADING",
-          isDeleted: false,
-          uploaderProjectId: projectId,
-          createdAt: { gt: new Date(Date.now() - UPLOADING_STALE_TIMEOUT) },
-        },
-      ],
-    },
-  });
+  const uploadedCount = await context.postgrest.client
+    .from("File")
+    .select("*", { count: "exact", head: true })
+    .eq("isDeleted", false)
+    .eq("uploaderProjectId", projectId)
+    .eq("status", "UPLOADED");
+
+  const uploadingCount = await context.postgrest.client
+    .from("File")
+    .select("*", { count: "exact", head: true })
```

Comment:
what means head: true? does it needed?

Reply:
> When set to `true`, `data` will not be returned. Useful if you only need the count.

Reply:
`.select("*", { count: "exact", head: true })` * looks suspicious I have a feeling that it generates something bad on sql side

Reply:
Better to see generated sql in postgres logs

Reply:
also where  .eq("uploaderProjectId", projectId)

Reply:
Looks like without head: true it loads data along with count